### PR TITLE
PE-2484: refactor: add new time formatter 

### PR DESCRIPTION
**Pull request by javdhu**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/4212391kfqoro
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/1v0594u09f1b0

### CODE REVIEW

File: lib/pages/shared_file/shared_file_side_sheet/shared_file_side_sheet.dart [hunk 1]
```diff
@@ -111,7 +112,7 @@ class _SharedFileSideSheetState extends State<SharedFileSideSheet> {
               Align(
                 alignment: Alignment.centerRight,
                 child: Text(
-                  yMMdDateFormatter.format(revisions.first.dateCreated),
+                  yMMdHmDateTimeFormatter.format(revisions.first.dateCreated),
```
Comment (matibat) [inline lib/pages/shared_file/shared_file_side_sheet/shared_file_side_sheet.dart:115]: We're missing to apply the change for "Date Created" on the Shared Page

**Review by matibat — COMMENTED** [review]
Working just fine! There's one single detail for the Shared Page

File: lib/pages/shared_file/shared_file_side_sheet/shared_file_side_sheet.dart (hunk 1 above)
Comment (matibat) [reply lib/pages/shared_file/shared_file_side_sheet/shared_file_side_sheet.dart:115]: Screen Shot 2022-11-14 at 13 59 28

**Review by matibat — APPROVED** [review]
LGTM!

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM!

**Review by karlprieb — APPROVED** [review]

**Review by matibat — APPROVED** [review]

**Review by karlprieb — APPROVED** [review]

**Pull request merged**